model/atmosphere/dycore/tests/dycore/integration_tests/test_velocity_advection.py

```diff
@@ -628,7 +628,9 @@ def test_compute_contravariant_correction_and_advection_in_vertical_momentum_equ
     horizontal_advection_of_w_at_edges_on_half_levels = savepoint_velocity_exit.z_v_grad_w()
     vertical_wind_advective_tendency = savepoint_velocity_init.ddt_w_adv_pc(istep_init - 1)
     contravariant_corrected_w_at_cells_on_model_levels = savepoint_velocity_init.z_w_con_c_full()
-    vertical_cfl = data_alloc.zero_field(icon_grid, dims.CellDim, dims.KDim, dtype=ta.vpfloat)
+    vertical_cfl = data_alloc.zero_field(
+        icon_grid, dims.CellDim, dims.KDim, dtype=ta.vpfloat, backend=backend
+    )  # TODO(edopao,havogt): read 'vertical_cfl' from savepoint
```

Comment:
```suggestion
    )  # TODO(edopao,havogt): or add 'vertical_cfl' to the savepoint
```
maybe more like this or ask @halungge what she thinks about that.

Reply:
I am not sure if it should be in the savepoint or not for the following reasons:
- pro savepoint: other output-only fields are there as well, e.g. `contravariant_correction_at_cells_on_half_levels`
- contra savepoint: it's an output-only field...